syntax = "proto3";

package objectstore;

message PullRequest {
  bytes object_id = 1;
  bytes puller_ip = 2;
}

message PullReply {
  bool ok = 1;
}

service ObjectStore {
  rpc Pull(PullRequest) returns (PullReply);
}
